.circleci/config.yml

```diff
@@ -168,6 +168,15 @@ caffe2_linux_build_defaults: &caffe2_linux_build_defaults
           cmake_args+=("-DBUILD_ATEN=ON")
         fi
 
+        if [[ $BUILD_ENVIRONMENT == py2-cuda9.0-cudnn7-ubuntu16.04 ]]; then
```

Comment:
I wonder whether this is the correct place to put `apt-get`. These kind of things are done at L32.

Reply:
Trying `setup_ci_environment` where we already have `apt-get update calls`..